rotkehlchen/tests/exchanges/test_ftx.py

```diff
@@ -561,6 +562,7 @@ def test_query_deposits_withdrawals_unexpected_data(mock_ftx: Ftx):
     )
 
 
+@pytest.mark.skipif('CI' in os.environ, reason='FTX API is unreachable after the incident')
```

Comment:
"the incident"

![giphy (6)](https://user-images.githubusercontent.com/1658405/201976598-94dfaca3-fda4-4a0b-ab2e-dbc250cd5a59.gif)